### PR TITLE
DOCSP-25031 clone diagrams

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Creating the Clone Diagrams task page

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/docs-relational-migrator/DOCSP-25031/diagrams/manage-diagrams/clone-diagrams/

## JIRA

https://jira.mongodb.org/browse/DOCSP-25031

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=659c2748dcc0144a513013a7

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/diagrams/manage-diagrams/clone-diagrams.txt [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
+
+      .. image:: /img/diagrams/diagram_properties.png
+         :alt: Diagram Properties pane
+
+      The :guilabel:`Diagram Properties` pane displays the active diagram name and
+      notes. 
+
+   .. step:: Click the :icon-fa5:`ellipsis-h` button in the :guilabel:`Diagram Properties` pane
+
+      A menu opens.
```
Comment (nvillahermosa-mdb) [inline source/diagrams/manage-diagrams/clone-diagrams.txt:53]: Suggest putting this line after the image to stay consistent.

File: source/diagrams/manage-diagrams/clone-diagrams.txt [hunk 2]
```diff
@@ -1,13 +1,68 @@
...
+   .. step:: Click the :icon-fa5:`ellipsis-h` button in the :guilabel:`Diagram Properties` pane
+
+      A menu opens. 
+
+      .. image:: /img/diagrams/duplicate_diagram.png
+         :alt: Diagram Properties Options menu
+
+   .. step:: Click the Duplicate Diagram option
+
+      This create and opens a new diagram with the same entities as the
```
Comment (nvillahermosa-mdb) [inline source/diagrams/manage-diagrams/clone-diagrams.txt:60]:
```suggestion
      This creates and opens a new diagram with the same entities as the
```

File: source/diagrams/manage-diagrams/clone-diagrams.txt [hunk 3]
```diff
@@ -1,13 +1,68 @@
...
+      duplicated diagram. The visibility of entities on your new diagram
+      is identical to the visibility of entities on the duplicated
```
Comment (nvillahermosa-mdb) [inline source/diagrams/manage-diagrams/clone-diagrams.txt:62]:
```suggestion
      is identical to the duplicated
```

**Review by nvillahermosa-mdb — COMMENTED** [review]
LGTM with a couple of minor suggestions. Thanks!

File: source/diagrams/manage-diagrams/clone-diagrams.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/clone-diagrams.txt:53]: Done!

File: source/diagrams/manage-diagrams/clone-diagrams.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/clone-diagrams.txt:60]: Done (rephrased re Ashley's comment on https://github.com/mongodb/docs-relational-migrator/pull/81 to " A new diagram with the same entities...."

File: source/diagrams/manage-diagrams/clone-diagrams.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/clone-diagrams.txt]: Done! Thanks for your feedback!

**Review by satyasinha — APPROVED** [review]
nice 👍

**Pull request merged**